### PR TITLE
[PRM-1571] - Updated LPP2 crystallised and estimated view

### PR DESCRIPTION
**Pull request by LewisMitchell**
Needs AT update
Trader estimate: 
trader-estimate
Posted: 
posted
Agent estimate:
agent-estimate

### CODE REVIEW

**Comment by ruthdas03** [comment]
ATs run ok against this branch (estimate and crystallised view). Are we considering substituting the tests desription having word 'additional' with 'Second'.

**Comment by LewisMitchell** [comment]
> ATs run ok against this branch (estimate and crystallised view). Are we considering substituting the tests desription having word 'additional' with 'Second'.

Where is the 'additional' wording?

File: test/views/CalculationViewSpec.scala [hunk 1]
```diff
@@ -48,18 +48,21 @@ class CalculationViewSpec extends SpecBase with ViewBehaviours with ViewUtils {
...
     val thirtyDayCalculation = "#30-day-calculation"
+
+    val whenPenaltyIncreases = "#when-it-increases"
+
+    val calculation = "#calculation"
   }
 
   "CalculationView" should {
 
     "if it is an additional penalty and the penalty is estimated" must {
```
Comment (ruthdas03) [inline test/views/CalculationViewSpec.scala:59]: So in the above test description we have"if it is an additional penalty and the penalty is estimated" can this be "if it is second late payment penalty and the penalty is estimated"

File: test/views/CalculationViewSpec.scala [hunk 2]
```diff
@@ -105,11 +104,10 @@ class CalculationViewSpec extends SpecBase with ViewBehaviours with ViewUtils {
     "if it is an additional penalty and the penalty is not estimated" must {
```
Comment (ruthdas03) [inline test/views/CalculationViewSpec.scala:104]: Similarly here "if it is an additional penalty and the penalty is not estimated" can be "if it is second late payment penalty and the penalty is not estimated"

File: test/views/CalculationViewSpec.scala [hunk 3]
```diff
@@ -300,5 +296,77 @@ class CalculationViewSpec extends SpecBase with ViewBehaviours with ViewUtils {
 
       behave like pageWithExpectedMessages(expectedContent)(docWithOnlyOneCalculation)
     }
+
+    "it is an additional penalty and the user is an Agent (no estimate)" must {
```
Comment (ruthdas03) [inline test/views/CalculationViewSpec.scala:300]: "it is an additional penalty and the user is an Agent (no estimate)" can be "it is second payment penalty and the user is an Agent (no estimate)"

File: test/views/CalculationViewSpec.scala [hunk 4]
```diff
@@ -300,5 +296,77 @@ class CalculationViewSpec extends SpecBase with ViewBehaviours with ViewUtils {
...
+        Selector.summaryListRowValue(3) -> "£40.10",
+        Selector.summaryListRowKey(4) -> th4LPP,
+        Selector.summaryListRowValue(4) -> "£10.40",
+        Selector.link -> link
+      )
+
+      behave like pageWithExpectedMessages(expectedContent)
+    }
+
+    "it is an additional penalty and the user is an Agent (estimate)" must {
```
Comment (ruthdas03) [inline test/views/CalculationViewSpec.scala:336]: "it is an additional penalty and the user is an Agent (estimate)" can be "it is a second payment penalty and the user is an Agent (estimate)"

**Review by ruthdas03 — APPROVED** [review]

**Pull request merged**